schemas/json/examples/ReferencesJsonExample.json

```diff
@@ -0,0 +1,34 @@
+{
+  "submodels":[
+    {
+      "modelType":{
+        "name":"Submodel"
+      },
+      "idShort":"S1",
+      "identification":{
+        "id":"http://env.com/submodels/S1",
+        "idType":"IRI"
+      },
+      "submodelElements":[]
+    }
+  ],
+  "conceptDescriptions":[],
+  "assetAdministrationShells":[
+    {
+      "modelType":"AssetAdministrationShell",
```

Comment:
Which variant is correct: the one with name attribute or without?

     "modelType":"AssetAdministrationShell",

Or

      "modelType":{
        "name":"Submodel"
      },

Reply:
I just took the example from the DotAAS v3.1 and did not make any customization.

Reply:
@Manu3756 May you please have look. It seems to be a bug in V3.0RC01, both variants are used in the examples but I assume only one of them is correct.

Which variant is correct: the one with name attribute or without?

 "modelType":"AssetAdministrationShell",

Or

  "modelType":{
    "name":"Submodel"
  },

Reply:
In the example from the book, the exact line "modelType" is: "AssetAdministrationShell".